### PR TITLE
feedback-delayed_deadline

### PR DESCRIPTION
**Pull request by mingxue-ordering**
story: https://app.asana.com/0/1200550251832819/1202141091343461

### CODE REVIEW

File: example/themes/business/src/components/OrdersOption/index.tsx [hunk 1]
```diff
@@ -929,47 +897,28 @@ export const StatusBlock = (props: any) => {
...
-      <OText color={theme.colors.disabled}>:</OText>
-      <TextInput
-        placeholder='MM'
-        keyboardType='number-pad'
-        maxLength={2}
-        style={{ ...styles.inputStyle, width: 40 }}
-        onChangeText={minute => handleChangeInput(minute, 'minute')}
-      />
+      <OText style={styles.settingTime} color={theme.colors.disabled}>{configs?.order_deadlines_delayed_time?.value}</OText>
+      <OText>min</OText>
```
Comment (carlosq-ordering) [inline example/themes/business/src/components/OrdersOption/index.tsx:921]: Don't hardcode the text

**Review by carlosq-ordering — CHANGES_REQUESTED** [review]
Please provide somes captures, images or vids, thanks

**Comment by mingxue-ordering** [comment]
Demo: https://d.pr/i/bYsyxj

**Comment by mingxue-ordering** [comment]
> Please provide somes captures, images or vids, thanks

Demo: https://d.pr/i/bYsyxj

**Review by carlosq-ordering — APPROVED** [review]

**Pull request merged**